web/controllers/BlogPostController.php

```diff
@@ -61,12 +64,19 @@ public function actionIndex()
         $models = $searchResult->offset($pages->offset)
         ->limit(10)
         ->all();
+        $userData = array();
```

Comment:
Pošto je novi Yii i nova verzija php-a.. Počni se tjerat da koristiš `[]` umjesto `array()`. Više kao savjet.